src/app/api/tasks/tasks.service.ts

```diff
@@ -307,6 +307,8 @@ export class TasksService extends BaseService {
       where: { assigneeId, assigneeType, workspaceId: this.user.workspaceId },
     })
     await this.db.label.deleteMany({ where: { label: { in: labels } } })
+
+    return tasks
```

Comment:
Shouldn't we refresh the tasks after delete?

Reply:
No, I intended for the var to contain a copy of the old deleted tasks @priosshrsth